### ISSUE TITLE
Won’t build on Windows

### ISSUE THREAD

**Issue by simonjwright** [simonjwright/cortex-gnat-rts#11]
The RTSs won’t build on Windows: reports
```
   [C]            adaint.c
   [Ada]          s-gearop.adb
fatal error, run-time library not installed correctly
cannot locate file system.ads
compilation abandoned
gprbuild: *** compilation phase failed
```

**Comment by simonjwright** [comment]
The problem is that the file `<rts>/ada_source_path` has colon `:` path separators; on Windows that needs to be semicolon `;` (or, portably, newline :-)

_gprbuild_ and _gnatmake_ are happy with either separator, but it looks as though this issue is in the compiler itself.

**Comment by simonjwright** [comment]
Fixed in ee772ce.
